Core/Core/Features/Assignments/AssignmentDetails/AssignmentDetailsView.swift

```diff
@@ -259,7 +259,8 @@ public struct AssignmentDetailsView: View, ScreenViewTrackable {
             launchType: "assessment",
             isQuizLTI: assignment.first?.isQuizLTI,
             assignmentID: assignmentID,
-            from: controller.value
+            from: controller.value,
+            env: .shared
```

Comment:
This view receives an environment in its initializer. Shouldn't we use that? It also has the `@Environment(\.appEnvironment)` property added and that is used in the view. Kind of confusing having a 3rd variable used here.